assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+    "dgc-getting-started-osx-1-2": "完成下載後，在下拉選單中的\"下載\"項目，開啟檔案。",
+    "dgc-getting-started-osx-1-3": "下載 MacOS 版本 MultiDoge",
+    "dgc-getting-started-osx-2-1": "點擊檔案後會出現新視窗，拖曳 \"MultiDoge\" 圖示到 \"Applications\" 資料夾中。",
+    "dgc-getting-started-osx-2-2": "這將移動您的錢包到 Applications 列表",
+    "dgc-getting-started-osx-acknowledge": "MacOS與圖片創建者 <a href=\"https://www.reddit.com/user/FlippyDog101\" target=\"_blank\">/u/FlippyDog101</a>. 網頁創建者 <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-linux-1-1": "點選下方連結下載狗狗幣錢包 MultiDoge <br/>儲存下載檔案",
+    "dgc-getting-started-linux-1-2": "找尋 (.jar) 利用您的檔案管理員打開程式",
+    "dgc-getting-started-linux-1-3": "下載 Linux 版本 MultiDoge",
+    "dgc-getting-started-linux-2-0": "２、設定權限",
+    "dgc-getting-started-linux-2-1": "右鍵點選 (.jar) 檔案後，選擇 \"Properties\".",
+    "dgc-getting-started-linux-2-2": "前往 \"Permissions\" 分頁，確認 \"Execute\" 權限允許執行檔案，並關閉 properties 視窗。",
+    "dgc-getting-started-linux-3-0": "３、執行 OpenJDK Java!",
+    "dgc-getting-started-linux-3-1": "再一次右鍵點選 (.jar) 檔案, 滑鼠移置 \"Open With\"，然後選擇 OpenJDK Java 6 or 7 Runtime.",
+    "dgc-getting-started-linux-3-2": "Okay, we might've lied about 1, 2, 3 on Linux. But, we need to explain more compared to other Operating Systems. Sorry!",
+    "dgc-getting-started-linux-4-0": "4、安裝錢包",
+    "dgc-getting-started-linux-4-1": "在安裝過程中，當您被詢問許可協議請接受並繼續安裝。",
+    "dgc-getting-started-linux-4-2": "完成安裝後，找到 MultiDoge 捷徑並開啟。",
+    "dgc-getting-started-linux-5-0": "５、使用錢包！",
+    "dgc-getting-started-linux-5-1": "啟動後，您的錢包將會在左下方出現等待進度條，您必須等待 1 ~ 5 分鐘讓錢包同步。",
+    "dgc-getting-started-linux-5-2": "錢包與區塊鏈網路同步後，您就可以開始使用狗狗幣錢包了！",
+    "dgc-getting-started-linux-acknowledge": "Linux 指引與圖片創建者 <a href=\"https://www.reddit.com/user/BloodyMonday\" target=\"_blank\">/u/BloodyMonday</a>. 網頁創建者 <a href=\"https://reddit.com/user/PowerLemons\" target=\"_blank\">/u/PowerLemons</a>.",
+    "dgc-getting-started-android-1-1": "Go to the Play Store and search \"Dogecoin Wallet\" on the Play Store, or click the \"Get it on Google Play\" button.",
+    "dgc-getting-started-android-1-2": "Press the \"Install\" button. Press the \"Accept\" button on the App Permission Requests panel.",
+    "dgc-getting-started-android-1-3": "Wait for the application to download and install.",
+    "dgc-getting-started-android-2-0": "2. Launch The Wallet.",
+    "dgc-getting-started-android-2-1": "On the same page, press the \"Open\" button. You will be directed to the Dogecoin Wallet.",
+    "dgc-getting-started-android-2-2": "You can also access the Dogecoin Wallet within the app drawer on your Android™ smartphone.",
+    "dgc-getting-started-android-3-1": "Now you have a full Dogecoin wallet in your hand!",
+    "dgc-getting-started-android-3-2": "On the top, you have your Address Book and Settings. <br/>Under it, you have your Dogecoin address on the left, Dogecoin address in the QR code on the right, and your Dogecoin balance.",
```

```suggestion
    "dgc-getting-started-android-3-2": "在畫面的上方為您的位址簿以及設定按鈕。 <br/>在其下方，左邊為您的狗狗幣地址、右邊為您的二維條碼以及狗狗幣餘額。",
```
The app show Address Book as "位址簿", however by this document translation standard should be "地址簿" .
By the way, dogecoin address is no longer showing on the left side.